common/src/main/java/org/apache/rocketmq/common/MixAll.java

```diff
@@ -87,6 +87,8 @@ public class MixAll {
     public static final String REPLY_MESSAGE_FLAG = "reply";
     public static final String LMQ_PREFIX = "%LMQ%";
     public static final String MULTI_DISPATCH_QUEUE_SPLITTER = ",";
+    public static final String MULTI_SEPARATE = "//";
+    public static final String SEPARATE_DOT = "/.";
```

Comment:
IMO, maybe someone use hide file as acl config file. BTW, can we use regular expression to check whether acl config file path is legal or not, instead of changing file path?

Reply:
1. i can only judge the last 2 char 
2.  what you describe  is when acl config file path is illegal( contain “//” or “ /.” ) ,throw AclException directly?